teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure to use `=` for the name parameter 
```suggestion
  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
```